src/checks/fastlyKeyExpiration.check.js

```diff
@@ -14,6 +15,7 @@ class FastlyKeyExpirationCheck extends Check {
 	constructor(options) {
 		super(options);
 		this.fastlyKey = options.fastlyKey;
+		this.panicGuide options.panicGuide || defaultPanicGuide;
```

Comment:
I think just missing an equals here: `this.panicGuide =`

Reply:
Yep, I realized last moments of the work shift I was faster than lucid 🤣